### PR TITLE
Use Kodi GUI language instead of system locale

### PR DESCRIPTION
**Pull request by CGarces**
In some cases system locale <> Kodi GUI Language.

In other cases like LibreELEC the OS itself not provide by default a locale information.

Tested on Debian with locale es_ES.UTF-8 and Kodi in English.

### CODE REVIEW

**Comment by Varstahl** [comment]
As you said, `xbmc.getLanguage(xbmc.ISO_639_1, False)` returns the UI language, although we would need the full locale so `getLanguage(xbmc.ISO_639_1, True)`. However, that's quite an arbitrary choice.

Several people I know of run various software with the english UI, for localization problems and whatnot, although they get the content served in whatever language they want. So, in my case, it would be English UI and Italian audio/subs. So, given the context, it would be better to get the default audio language locale as a more sensible option, rather than the UI itself, as it would more appropriately match the user's expectations. Still, that's arbitrary too.

So, given that the language is currently only used for `Accept-Language`, and that amazon completely ignores them (except for the fact that they're required), I'd vote to keep it as is, to mimic normal browser behaviour. Being as useless as it is, given that `Accept-Language: Klingon` or `Accept-Language: AMZ-Y-U-DO-DIS` are perfectly valid, it would also make no sense giving the user an option choice for something that doesn't even matter.

As it is now, at best it keeps being ignored from amazon, and so it's useless code running for no reason, at worst if amazon changes something in the data returned from the server, the results would match the expectancy.

**TL;DR:** I'm not too keen on accepting this as-is, but I'm open to hear the reason why we should change.

**Comment by CGarces** [comment]
Hi.
The PR was due a misinterpretation from my side.
I was assuming that  `Language = locale.getdefaultlocale()` was used to force the language returned by primevideo. But per your comment I see that not is true and now I'm completely lost.

If Amazon ignore the  Accept-Language, why that lines of code are needed? 
Why noy just use `userAcceptLanguages = 'en-gb, en;q=0.5'` and remove the `Language` variable to avoid confusion?

How is determined the language returned by  primevideo? by the user setting inside the primevideo account?

**Comment by Varstahl** [comment]
> If Amazon ignore the Accept-Language, why that lines of code are needed?

They aren't, per se. In the original `getURL`, since amazon expects (probably for logging purposes) to receive the `Accept-Language`, a default string was used, `de_DE, en;q=0.5`. I introduced the locale code when I was expecting to localize parts of the addon into the user's locale, but in hindsight it quite makes no sense. I then kept it to parse the dates, but that wouldn't work either. And so here we are now.

> Why noy just use userAcceptLanguages = 'en-gb, en;q=0.5' and remove the Language variable to avoid confusion?

It's a pretty silly answer, but it's an answer nonetheless: I've kept it because it was already there. It does no harm, it costs next to nothing in terms of CPU power, and in case something changes in the future we're already providing a valid `Accept-Language`, so even if amazon changes how the backend works, it will still work for us.

> How is determined the language returned by primevideo? by the user setting inside the primevideo account?

Yes, the `lc-main-av` is what is used by amazon to give back the titles/description/web. I'm not sure if the other API endpoints we use actually do something with the `Accept-Language` we provide, but since we only use numbers and ASINs from the catalog information API, even there it makes no difference at the moment.

**Comment by CGarces** [comment]
So this PR is useless.
